apps/backend/src/services/compose_deployment.py

```
@@ -377,38 +381,40 @@ async def _update_volume_paths(
                     updated_volumes.append(volume)
-            
+
             if updated_volumes != volumes:
                 service_config['volumes'] = updated_volumes
                 result.volume_updates[service_name] = service_volume_updates
 
     async def _assign_ports(
         self,
-        compose_data: Dict[str, Any],
+        compose_data: dict[str, Any],
         device_config: Device,
```

Ruff (0.12.2)

Unused method argument: `device_config`

(ARG002)